parse-tgfd.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import sys
+import struct
+
+with open(sys.argv[1], 'rb') as in_file:
+  data = in_file.read()
+  
+  if len(data) == 1408:
+    is_console = True # Dreamcast
+    print("Non-PC files are not supported at this point")
+    sys.exit(1)
+  elif len(data) == 4056:
+    is_console = False # PC Version
+    # FIXME: Assert the header is correct
+    data = data[4:]
+  else:
+    print("Unsure what file this is")
+    sys.exit(1)
+
+  tournaments = [
+    "Amateur Podracing Circuit",
+    "Semipro Podracing Circuit",
+    "Galactic Podracing Circuit",
+    "Invitational Podracing Circuit"
+  ]
+
+  races = [
+    "Boonta Training Course", # 3 laps = 0, best lap = 50 + 0
+    "Mon Gazza Speedway", # best lap = 50 + 32
+    "Beedo's Wild Ride", # 3 laps = 4, best lap = 50 + 4
+    "Aquilaris Classic",
+    "Malastare 100",
+    "Vengeance",
+    "Spice Mine Run",
+    "<Unused 7>",
+
+    "Sunken City", # best lap = 50 + 14
+    "Howler Gorge", # best lap = 50 + 6
+    "Dug Derby",
+    "Scrapper's Run",
+    "Zugga Challenge",
+    "Baroo Coast",
+    "Bumpy's Breakers",
+    "<Unused 15>",
+
+    "Executioner", # 3 laps = 40, best lap = 50 + 40
+    "Sebulba's Legacy",
+    "Grabvine Gateway",
+    "Andobi Mountain Run",
+    "Dethro's Revenge", # best lap = 50 + 20
+    "Fire Mountain Rally",
+    "The Boonta Classic",
+    "<Unused 23>",
+
+    "Ando Prime Centrum",
+    "Abyss",
+    "The Gauntlet",
+    "Inferno", # best lap = 50 + 30
+    "<Unused 28>",
+    "<Unused 29>",
+    "<Unused 30>",
+    "<Unused 31>"
+  ]
```

Comment:
There might be a different table for track besttimes as I can't find a proper pattern.
Note that all best times currently have an even index.